components/brave_ads/browser/notification_helper_android.cc

```diff
@@ -73,13 +72,14 @@ NotificationHelper* NotificationHelper::GetInstanceImpl() {
 
 ///////////////////////////////////////////////////////////////////////////////
 
-bool NotificationHelperAndroid::IsBraveAdsNotificationChannelEnabled() const {
+bool NotificationHelperAndroid::IsBraveAdsNotificationChannelEnabled(bool foregroundChannel) const {
```

Comment:
`const bool`

Reply:
pass by value: doesn't have to be `const`

Reply:
See below